src/Command/PelagosClearErrorAndFileCommand.php

```diff
@@ -73,9 +85,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             try {
                 $delId = $fileToDelete->getId();
                 $this->entityManager->remove($fileToDelete);
-                @unlink($fileToDelete->getPhysicalFilePath());
+                $uploadDir = preg_quote($this->homedirPrefix . '/upload/', '/');
+                if (preg_match("/$uploadDir/", $fileToDelete->getPhysicalFilePath())) {
+                    $physicalFileToDelete = $fileToDelete->getPhysicalFilePath();
+                } else {
+                    $physicalFileToDelete = $this->dataStoreDirectory . '/' . $fileToDelete->getPhysicalFilePath();
+                }
+                @unlink($physicalFileToDelete);
```

Comment:
This will directly delete a file from the Datastore. Instead inject the Datastore util, and do exactly what DeleteFileHandler does. Datastore also will have those paths already injected.